src/ansys/sherlock/core/parts.py

```diff
@@ -929,3 +930,89 @@ def update_parts_list_properties(self, project, cca_name, part_properties):
             for error in e.str_itr():
                 LOG.error(error)
             raise e
+
+    def export_net_list(
+        self,
+        project,
+        cca_name,
+        output_file,
+        col_delimiter=TableDelimiter.COMMA,
+        overwrite_existing=False,
+        utf8_enabled=False,
+    ):
+        """Export a net list to a delimited output file.
+
+        Parameters
+        ----------
+        project : str
+            Name of the Sherlock project.
+        cca_name : str
+            Name of the CCA.
+        output_file : str
+            Full path for the output file where the net list will be written.
+        col_delimiter : TableDelimiter, optional
+            The delimiter character to be used. Defaults to TableDelimiter.COMMA.
+        overwrite_existing : bool, optional
+            Flag to determine if existing files should be overwritten if they match the output_file.
+            Defaults to False.
+        utf8_enabled : bool, optional
+            Flag that specifies if UTF-8 will be used. Defaults to False.

```

Comment:
For `overwrite_existing` and `utf8_enabled`, I think the comments should indicate that the parameter is only applicable for .CSV files.